spec/models/district_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+describe District, type: :model do
+	it { should have_and_belong_to_many(:users) }
+end
```

Comment:
Please add blank line at end of file, and make sure this file uses two spaces not tabs.